invenio_deposit/models.py

```diff
@@ -52,3 +52,8 @@ class DepositBucket(db.Model):
     )
 
     bucket = db.relationship(Bucket)
+    record = db.relationship(
```

Comment:
are you sure about that?
Because I think you will find this relationship in all kind of records (also if it's not a deposit).